src/eslintPlugins.ts

```diff
@@ -154,6 +155,7 @@ const plugins = [
   ["lit", litRules],
   ["lodash", lodashRules],
   ["lodash-fp", lodashFpRules],
+  ["lwc", lwcRules],
```

Comment:
This must be:
@lwc/lwc
So that it can then call the proper module:
@lwc/eslint-plugin-lwc
And thus should be placed alphabetically with the remaining plugins starting with "@".